src/lando/main/models/commit_map.py

```diff
@@ -0,0 +1,90 @@
+import requests
+from django.db import models
+
+from lando.main.models import BaseModel
+from lando.main.models.repo import Repo
+
+
+class CommitMap(BaseModel):
+    """Map a git hash to an hg hash, based on a specific repo."""
+
+    HGMO_PUSHLOG_TEMPLATE = "https://hg.mozilla.org/{}/json-pushes"
+    REPO_MAPPING = (("firefox", "mozilla-unified"),)
+
+    git_hash = models.CharField(default="", unique=True)
+    hg_hash = models.CharField(default="", unique=True)
```

Comment:
I think we need the unique constraints to cover the git_repo (git_repo first, then hash).

Reply:
Ah, also, we know the length of a commit SHA, so we could limit the column size. I'd also suggest not having a defaultand, as we'd run into the uniqueness constraint issue again.